consoleme/lib/self_service/models.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+from typing import List, Optional
+
+from consoleme.lib.pydantic import BaseModel
+
+
+class SelfServiceResourceType(Enum):
+    AwsIamRole = "AwsIamRole"
+    HoneybeeAwsIamRoleTemplate = "HoneybeeAwsIamRoleTemplate"
```

Comment:
This isn't OSS-friendly, but I'm not sure there's a better way to handle it at the moment

Reply:
Yeah, would love to OSS Honeybee in the long-term, I believe it's okay for these references to be OSS. Plus it confuses the bug bounty hunters (We're watching you!) :-P.

Reply:
👀